data/en/get_started.json

```diff
@@ -364,16 +504,16 @@
               "description": "Download a standalone zip file for trying out CDAP on your Linux or Mac OSX machine.",
               "img": "/images/get-started/linux-mac.png",
               "button": {
-                "buttonTitle": "CDAP 6.0.0 for Linux/Mac",
```

Comment:
@MEseifan should we be updating this or adding a brand new item object inside the ``items`` list?

Reply:
I think these need to be updated because I dont see other items in this list from previous versions